File: Search/ECMSearchWPF/app_code/clsPreviewContent.vb
' ***********************************************************************
' Assembly         : ECMSearchWPF
' Author           : wdale
' Created          : 06-28-2020
'
' Last Modified By : wdale
' Last Modified On : 06-28-2020
' ***********************************************************************
' <copyright file="clsPreviewContent.vb" company="D. Miller and Associates, Limited">
'     Copyright @ DMA Ltd 2020 all rights reserved.
' </copyright>
' <summary></summary>
' ***********************************************************************

''' <summary>
''' Class clsPreviewContent.
''' </summary>
Public Class clsPreviewContent
    ''' <summary>
    ''' Previews the document.
    ''' </summary>
    Public Sub PreviewDoc()
        'Dim uri As Uri = New Uri("http://www.ecmlibrary/$Preview/LCSE-900010Firmware Support Manual.PDF", UriKind.RelativeOrAbsolute)
        Try
            System.Diagnostics.Process.Start("http://www.ecmlibrary/$Preview/LCSE-900010Firmware Support Manual.PDF")
        Catch
            'Code to handle the error.
        End Try

    End Sub
End Class

File: Search/ECMSearchWPF/CS/app_code/clsPreviewContent.cs
// VBConversions Note: VB project level imports
using System.Windows.Input;
using System.Windows.Data;
using System.Windows.Documents;
using System.Xml.Linq;
using System.Collections.Generic;
using System.Windows.Navigation;
using System.Windows.Media.Imaging;
using System.Windows;
using Microsoft.VisualBasic;
using System.Windows.Media;
using System.Collections;
using System;
using System.Windows.Shapes;
using System.Windows.Controls;
using System.Threading.Tasks;
using System.Linq;
using System.Diagnostics;
// End of VB project level imports



namespace ECMSearchWPF
{
	public class clsPreviewContent
	{
		public void PreviewDoc()
		{
			//Dim uri As Uri = New Uri("http://www.ecmlibrary/$Preview/LCSE-900010Firmware Support Manual.PDF", UriKind.RelativeOrAbsolute)
			try
			{
				System.Diagnostics.Process.Start("http://www.ecmlibrary/$Preview/LCSE-900010Firmware Support Manual.PDF");
			}
			catch
			{
				//Code to handle the error.
			}
			
		}
	}
	
}
